{"file": {"path": "/Users/dakixr/Desktop/github/scc/tmp-data/metadata/KnowledgeCaptureAndDiscoveryANDmintproject/wings-api/server/openapi_server/models/error.py", "fileNameBase": "error", "extension": "py"}, "dependencies": [{"from_module": "__future__", "import": "absolute_import", "type": "external"}, {"from_module": "datetime", "import": "date", "type": "external"}, {"from_module": "datetime", "import": "datetime", "type": "external"}, {"from_module": "typing", "import": "List", "type": "external"}, {"from_module": "typing", "import": "Dict", "type": "external"}, {"from_module": "openapi_server.models.base_model_", "import": "Model", "type": "external"}, {"from_module": "openapi_server", "import": "util", "type": "external"}], "classes": {"Error": {"doc": {"long_description": "Do not edit the class manually.", "short_description": "NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).", "full": "NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).\n\nDo not edit the class manually."}, "extend": ["Model"], "min_max_lineno": {"min_lineno": 12, "max_lineno": 95}, "methods": {"__init__": {"doc": {"short_description": "Error - a model defined in OpenAPI", "args": {"code": {"description": "The code of this Error.  # noqa: E501"}, "message": {"description": "The message of this Error.  # noqa: E501"}}}, "args": ["self", "code", "message"], "min_max_lineno": {"min_lineno": 18, "max_lineno": 38}}, "from_dict": {"doc": {"short_description": "Returns the dict as a model", "args": {"dikt": {"description": "A dict."}}, "returns": {"description": "The Error of this Error.  # noqa: E501", "is_generator": false}}, "args": ["cls", "dikt"], "min_max_lineno": {"min_lineno": 39, "max_lineno": 49}, "calls": ["openapi_server.util.deserialize_model"]}, "code": {"doc": {"short_description": "Sets the code of this Error.", "args": {"code": {"description": "The code of this Error."}}}, "args": ["self", "code"], "min_max_lineno": {"min_lineno": 60, "max_lineno": 72}, "calls": ["ValueError"]}, "message": {"doc": {"short_description": "Sets the message of this Error.", "args": {"message": {"description": "The message of this Error."}}}, "args": ["self", "message"], "min_max_lineno": {"min_lineno": 83, "max_lineno": 95}, "calls": ["ValueError"]}}}}, "is_test": false}